node.gyp

```diff
@@ -583,6 +597,8 @@
             'src/node_crypto.h',
             'src/node_crypto_bio.h',
             'src/node_crypto_clienthello.h',
+            'src/node_crypto_clienthello-inl.h',
+            'src/node_crypto_groups.h',
```

Comment:
Are headers really necessary here at all?

Reply:
Sort of... they are added as dependencies for MSBuild and ninja. Also they are added to the MSVS GUI + intellisense + static analysis.
![image](https://user-images.githubusercontent.com/96947/49026643-5a630180-f16c-11e8-9b11-de246f2d6731.png)

Reply:
Maybe also relevant for XCode, and `compile_commands.json`